lib/utils/report.mjs

```diff
@@ -163,23 +160,15 @@ function isDisabled(ruleName, startLine, disabledRanges) {
 function isFixApplied({ fix, line, result: { stylelint }, ruleName }) {
 	const { disabledRanges, config = {}, fixersData } = stylelint;
 
-	if (!isFn(fix)) {
-		addFixData({ fixersData, ruleName, fixed: false });
-
-		return false;
-	}
+	if (!isFn(fix)) return addFixData({ fixersData, ruleName, fixed: false }), false;
```

Comment:
[suggest] Can you restore the original form? Because:

- `return addFixData(...)` looks like the `addFixData()` function would return something regardless nothing.
- The `return x, y` pattern is uncommon in this code base: let's keep it less surprising.

```suggestion
	if (!isFn(fix)) {
		addFixData({ fixersData, ruleName, fixed: false })

		return false;
	}
```

Reply:
> The `return x, y` pattern is uncommon…

Should we add an ESLint rule to our config to disallow it?
I liked the terseness.

Reply:
> Should we add an ESLint rule to our config to disallow it?

If we could add such an ESLint rule easily (e.g., a built-in rule), I'd be okay with it. But I don't have a strong hope since such a pattern is rare in this code base.

Reply:
https://eslint.org/docs/latest/rules/no-sequences

Reply:
Good. Can you open a PR to add the `no-sequences` rule to our ESLint config if you have time?
https://github.com/stylelint/eslint-config-stylelint/blob/f80257c6116306f7ee7e38457a6f7d98cffbe259/index.js

Reply:
Sorry, but this time I won't.
I actually like that pattern so you will understand that I don't want to initiate that change :)